app/models/site_banner.rb

```diff
@@ -0,0 +1,37 @@
+require 'rubygems'
+require 'json'
+
+class SiteBanner < ApplicationRecord
```

Comment:
Maybe I'm missing something, but is SiteBanner the right name here and elsewhere?  I'm thinking that this should be named PublisherBanner or BraveRewardsBanner since this isn't associated any site, and could be confusing since we use SiteChannelDetails, YoutubeChannelDetails, to differentiate channel type.